code/datums/trading/illegal.dm

```diff
@@ -0,0 +1,57 @@
+/datum/trader/ship/illegal
+	name = "Unknown"
+	origin = "Unregistered Vessel"
+	typical_duration = 10
+
+/datum/trader/ship/illegal/get_item_value(var/trading_num)
+	. = ..()
+	. *= 1.5
+
+/datum/trader/ship/illegal/gunshop
+	speech = list(TRADER_HAIL_GENERIC    = "Welcome, have a look at our arnaments, just be quick.",
+				TRADER_HAIL_DENY         = "Our ship doesn't sell anything, you might've mistaken us for someone else.",
+
+				TRADER_TRADE_COMPLETE    = "Nice having business with you.",
+				TRADER_NO_BLACKLISTED   = "Not interested in that one.",
+				TRADER_NO_GOODS    = "Cash for guns, thats the deal.",
+				TRADER_NOT_ENOUGH  = "Looks you need money, more of it.",
+				TRADER_HOW_MUCH          = "That'd cost you VALUE.",
+
+				TRADER_COMPLEMENT_SUCCESS = "So, gonna buy anything?",
+				TRADER_COMPLEMENT_FAILURE   = "I don't think you understand how business works.",
+				TRADER_INSULT_GOOD       = "That's not how you do business, pal.",
```

Comment:
A successful insult should imply that they are acquiescing to you and lowering their prices.

Reply:
There are no successful insults.
"Insult_good" is for when your relations with the trader are not too low.

Reply:
Apologies, thought it meant something else.

Shadow does not do trading often.